appveyor.yml

```
@@ -2,6 +2,12 @@
 init:
   - git config --global core.autocrlf false
 
+install:
+  - sh: curl -sSL https://dot.net/v1/dotnet-install.sh | bash /dev/stdin --version 7.0.100
+  - sh: export DOTNET_ROOT=$HOME/.dotnet
+  - sh: export PATH="$PATH:$DOTNET_ROOT:$DOTNET_ROOT/tools"
+  - sh: dotnet --version
```

```suggestion
install:
  - sh: curl -sSL https://dot.net/v1/dotnet-install.sh | sudo bash /dev/stdin --verbose --install-dir /usr/share/dotnet --version 7.0.100
```

I am not sure if AppVeyor has a dedicated task to install dependencies (SDKs), but the above change will work.

---

Awesome! Thank you!